components/gallery/useGalleryItem.ts

```diff
@@ -14,6 +14,8 @@ const whichMimeType = async (data) => {
     return await getMimeType(sanitizeIpfsUrl(data.animation_url))
   } else if (data?.image) {
     return await getMimeType(sanitizeIpfsUrl(data.image))
+  } else if (data?.mediaUri) {
```

Comment:
remove else if's - for easier readabilty

```suggestion
const whichMimeType = async (data) => {
  if (data?.type) {
    return data?.type
  }

  if (data?.animation_url) {
    return await getMimeType(sanitizeIpfsUrl(data.animation_url))
  }
  if (data?.image) {
    return await getMimeType(sanitizeIpfsUrl(data.image))
  }
  if (data?.mediaUri) {
    return await getMimeType(sanitizeIpfsUrl(data.mediaUri))
  }

  return ''
}```